### PR TITLE
[wip] add olm e2e test

### PR DESCRIPTION
**Pull request by willkutler**
creating pr to test build on github

### CODE REVIEW

File: test/e2e-olm/e2e_suite_test.go [hunk 1]
```diff
@@ -0,0 +1,154 @@
+// Copyright (c) 2020 Red Hat, Inc.
```
Comment (ycao56) [inline test/e2e-olm/e2e_suite_test.go:1]: Remove copyright statement?

File: test/e2e-olm/e2e_suite_test.go [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+func TestE2e(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Policy Framework e2e Suite")
+}
+
+func init() {
+	klog.SetOutput(GinkgoWriter)
+	klog.InitFlags(nil)
+	flag.StringVar(&kubeconfigHub, "kubeconfig_hub", "", "Location of the kubeconfig to use; defaults to KUBECONFIG if not set")
+	flag.StringVar(&kubeconfigManaged, "kubeconfig_managed", "", "Location of the kubeconfig to use; defaults to KUBECONFIG if not set")
```
Comment (ycao56) [inline test/e2e-olm/e2e_suite_test.go:55]: Don't think you need two kubeconfig and kube clients. need some clean up

File: test/e2e-olm/e2e_suite_test.go [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+	By("Setup hub and managed client")
+	gvrPod = schema.GroupVersionResource{Version: "v1", Resource: "pods"}
+	gvrCRD = schema.GroupVersionResource{Group: "apiextensions.k8s.io", Version: "v1beta1", Resource: "customresourcedefinitions"}
+	gvrNS = schema.GroupVersionResource{Version: "v1", Resource: "namespaces"}
+	clientHub = NewKubeClient("", kubeconfigHub, "")
+	clientHubDynamic = NewKubeClientDynamic("", kubeconfigHub, "")
+	clientManaged = NewKubeClient("", kubeconfigManaged, "")
+	clientManagedDynamic = NewKubeClientDynamic("", kubeconfigManaged, "")
+	defaultImageRegistry = "quay.io/open-cluster-management"
+	defaultImagePullSecretName = "multiclusterhub-operator-pull-secret"
```
Comment (ycao56) [inline test/e2e-olm/e2e_suite_test.go:69]: need clean up

File: test/e2e-olm/e2e_suite_test.go [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+func TestE2e(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Policy Framework e2e Suite")
+}
+
+func init() {
+	klog.SetOutput(GinkgoWriter)
+	klog.InitFlags(nil)
+	flag.StringVar(&kubeconfigManaged, "kubeconfig_managed", "", "Location of the kubeconfig to use; defaults to KUBECONFIG if not set")
```
Comment (ycao56) [inline test/e2e-olm/e2e_suite_test.go:47]: I suggest to rename this to just `kubeconfig` as there is only one kind cluster

File: test/e2e-olm/e2e_suite_test.go [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+
+}
+
+var _ = BeforeSuite(func() {
+	By("Setup hub and managed client")
+	gvrPod = schema.GroupVersionResource{Version: "v1", Resource: "pods"}
+	gvrCRD = schema.GroupVersionResource{Group: "apiextensions.k8s.io", Version: "v1", Resource: "customresourcedefinitions"}
+	clientManaged = NewKubeClient("", kubeconfigManaged, "")
```
Comment (ycao56) [inline test/e2e-olm/e2e_suite_test.go:55]: same as above

File: test/e2e-olm/e2e_suite_test.go [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/klog/v2"
+)
+
+var (
+	userNamespace         string
+	clusterNamespace      string
```
Comment (ycao56) [inline test/e2e-olm/e2e_suite_test.go:23]: some variables are not being used

**Comment by willkutler** [comment]
closing because of https://github.com/gatekeeper/gatekeeper-operator/pull/92

**Pull request closed without merging**